Equilobe.DailyReport.Models/BitBucket/State.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Models.BitBucket
+{
+    public enum State
+    {
+        MERGED,
```

Comment:
is it necessary to use capital letters for all of them?

Reply:
this is the exact string received from the request

Reply:
the Parse method has an overload, and you can tell it to ignore the casing